CDP4CrossViewEditor/ViewModels/ParameterTypeSelectorViewModel.cs

```diff
@@ -165,5 +198,50 @@ protected override void ExecuteMoveToTarget()
         {
             ExecuteMove(this.ParameterTypeSourceList, this.ParameterTypeTargetList, this.SelectedSourceList);
         }
+
+        /// <summary>
+        /// Move power related parameter types to target list
+        /// </summary>
+        private void ExecuteMovePower()
+        {
+            var powerParameterType = new List<ParameterType>();
+            var iterationElements = this.Iteration.Element.AsEnumerable<ElementBase>();
+
+            if (iterationElements == null)
+            {
+                return;
+            }
+
+            var elements = iterationElements.Union(this.Iteration.Element.SelectMany(e => e.ContainedElement).AsEnumerable<ElementBase>());
```

Comment:
All the elements/parameters we care about and we should iterate are the ones displayed in the grid.